app/controllers/users_controller.rb

```diff
@@ -67,28 +67,48 @@ def list_pending_requested
     @roles = Role.all
   end
 
+
+  # Differences between show _selection and show:
+  #   The key difference between show_selection and the show methods is in how they determine whether the current user has the authority to delete/edit a selected user
+  #   The show method only checks if the current user is not a student or him/herself. It assumes that all other roles are authorised to edit/delete the information of all users.
+  #   The show_selection method allows access only if the current user is higher up the hierarchy of roles than the user s/he is requesting to edit.
+  # The show method additionally calculates the number of total users and the number of groups the user is participating in.
+  # These numbers are used in the show view to display appropriate warning messages when deleting a user.
```

Comment:
You probably don't have to indicate these points (lines 71-76) as the "key differences", as they can be inferred from the code. Instead, it might be more useful to point out the locations from where the `show_selection` method is **called**, versus the locations from where the `show` method is called. This way, we will be able to figure out why these two methods were created and structured the way they are and perhaps be able to refactor them if necessary.

Reply:
Will be fixed in the next update